### PR TITLE
Remove command option - ApiServerOption

### PR DESCRIPTION
**Pull request by hodovska**
- set built-in values for authorization config
	- add flag admission-control for specifying admission plugins

### CODE REVIEW

**Comment by aveshagarwal** [comment]
@hodovska could you look into using pkg/util/sets (sets.NewString etc) to make your code a bit more simplified?

**Comment by hodovska** [comment]
@aveshagarwal thanks for suggestion, now it is fixed

**Comment by ingvagabund** [comment]
@redhat-k8s-bot test please

File: integration-tests.sh [hunk 1]
```diff
@@ -65,47 +65,102 @@ echo ""
...
+
+
+echo ""
+echo ""
+echo "# Running with ResourceQuota admission plugin"
+echo "## Creating resource quota"
+$KUBECTL create resourcequota pod-quota --namespace=cluster-capacity --hard=pods=1
+
+$CC --podspec=examples/pod.yaml --resource-space-mode=ResourceSpacePartial --admission-control="ResourceQuota" | tee estimation.log
+if [ -z "$(cat estimation.log | grep 'Admission controller error')" ]; then
```
Comment (ingvagabund) [inline integration-tests.sh]: AdmissionControllerError is the error string

Comment (hodovska) [inline integration-tests.sh]: It depends on how many pods were scheduled (either zero or non-zero). This is the message, when I create a pod in cluster-capacity namespace before test run
```
# Running with ResourceQuota admission plugin
## Creating resource quota
resourcequota "pod-quota" created
Unable to create next pod to schedule: Admission controller error: pods "small-pod-0" is forbidden: exceeded quota: pod-quota, requested: pods=1, used: pods=1, limited: pods=1
```

Should we unify output message for both situations or expect them in test?:
 
```sh
...
if [ -z "$(cat estimation.log | grep 'Admission controller error\|AdmissionControllerError')"  ]; then
...
```

Comment (ingvagabund) [inline integration-tests.sh]: The error must be the same for both cases. Can you grep the place where the "Admission controller error" string is printed and replace it with "AdmissionControllerError"?

**Comment by hodovska** [comment]
@ingvagabund I updated the error message as you suggested.

File: cmd/cluster-capacity/app/options/options.go [hunk 2]
```diff
@@ -31,23 +31,24 @@ import (
...
 	"k8s.io/kubernetes/pkg/util/yaml"
 	schedopt "k8s.io/kubernetes/plugin/cmd/kube-scheduler/app/options"
 
 	"github.com/kubernetes-incubator/cluster-capacity/pkg/apiserver/cache"
-	"github.com/kubernetes-incubator/cluster-capacity/pkg/framework"
 	"github.com/kubernetes-incubator/cluster-capacity/pkg/framework/store"
 	"github.com/kubernetes-incubator/cluster-capacity/pkg/utils"
 )
 
+var SupportedAdmissionControllers = sets.NewString([]string{"LimitRanger", "ResourceQuota"}...)
```
Comment (aveshagarwal) [inline cmd/cluster-capacity/app/options/options.go]: I'd say rename it to defaultAdmissionControllers.

Comment (aveshagarwal) [inline cmd/cluster-capacity/app/options/options.go]: or may be just admissionControllersList.

Comment (aveshagarwal) [inline cmd/cluster-capacity/app/options/options.go]: ok nevermind, I think leave it as it SupportedAdmissionControllers. perhaps I now understand what it is trying to do.

**Comment by aveshagarwal** [comment]
LGTM.

**Comment by aveshagarwal** [comment]
@ingvagabund Do you have any other comments? Otherwise lets merge it.

**Comment by ingvagabund** [comment]
@redhat-k8s-bot test please

**Comment by ingvagabund** [comment]
travis-ci has the old config

**Comment by ingvagabund** [comment]
``Could not find .travis.yml, using standard configuration.``

**Comment by ingvagabund** [comment]
@hodovska can you just ``git commit --amend`` and ``git push --force``?

**Comment by ingvagabund** [comment]
@aveshagarwal if the RH CI passes, we can merge as well.

**Comment by ingvagabund** [comment]
```sh
./test.sh
# github.com/kubernetes-incubator/cluster-capacity/pkg/framework
pkg/framework/simulator_test.go:213: cannot use ApiServerOptions literal (type *ApiServerOptions) as type string in argument to New
ok  	github.com/kubernetes-incubator/cluster-capacity/pkg/apiserver	0.062s
ok  	github.com/kubernetes-incubator/cluster-capacity/pkg/framework/restclient	0.283s
FAIL	github.com/kubernetes-incubator/cluster-capacity/pkg/framework [build failed]
ok  	github.com/kubernetes-incubator/cluster-capacity/pkg/framework/strategy	0.075s
make: *** [test] Error 2
```

The error is still there.

**Comment by ingvagabund** [comment]
Should be enough to replace ``&ApiServerOptions{}`` with empty string in ``pkg/framework/simulator_test.go:213`` [1].

[1] https://github.com/kubernetes-incubator/cluster-capacity/blob/master/pkg/framework/simulator_test.go#L213

**Comment by ingvagabund** [comment]
@redhat-k8s-bot test please

**Pull request merged**